pkg/network/tracer/tracer.go

```diff
@@ -164,25 +159,14 @@ func NewTracer(config *config.Config) (*Tracer, error) {
 		sysctlUDPConnStreamTimeout: sysctl.NewInt(config.ProcRoot, "net/netfilter/nf_conntrack_udp_timeout_stream", time.Minute),
 		gwLookup:                   newGatewayLookup(config),
 		ebpfTracer:                 ebpfTracer,
-		closedConnsCh:              closedConnsCh,
 	}
-	tr.initClosedPolling()
 
-	return tr, nil
-}
+	err = ebpfTracer.Start(tr.closedCB)
+	if err != nil {
+		return nil, fmt.Errorf("could not start ebpf manager: %s", err)
```

Comment:
Do we need to tear anything down at this point, since the tracer object has been created along with possible HTTP and DNS features? I think this is why it was before object creation before.

Reply:
Great catch. I added some changes to handle that scenario